drivers/display/ssd1306.c

```diff
@@ -117,6 +117,13 @@ static inline int ssd1306_set_timing_setting(struct device *dev)
 static inline int ssd1306_set_hardware_config(struct device *dev)
 {
 	struct ssd1306_data *driver = dev->driver_data;
+	u8_t comPins;
+
+	if (DT_SOLOMON_SSD1306FB_0_HEIGHT == 32)
```

Comment:
The COM Pins Hardware Configuration is panel depended and is not depended on the panel height.
Please  add multiple display panel support to this driver (see SSD1673 driver).